backend/api/v1/endpoints/workout_day.py

```diff
@@ -0,0 +1,30 @@
+from db_models.models.custom_workout_day import CustomWorkoutDay
+from db_models.models.workout_day import WorkoutDay
+from rest_framework import serializers
+
+
+class WorkoutDaySerializer(serializers.ModelSerializer):
+
+    class Meta:
+        model = WorkoutDay
+        fields = (
+            'exercise',
+            'day_of_week',
+            'sets',
```

Comment:
what's this field for again? Thought it was an implicit field

Reply:
its implicit dw

Reply:
just for display.